aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncPage.java

```diff
@@ -0,0 +1,42 @@
+package com.amplifyframework.api.aws;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.api.ApiException;
+import com.amplifyframework.api.graphql.GraphQLOperation;
+import com.amplifyframework.api.graphql.GraphQLRequest;
+import com.amplifyframework.api.graphql.GraphQLResponse;
+import com.amplifyframework.api.graphql.Page;
+import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.Consumer;
+
+public class AppSyncPage<T> extends Page<T> {
```

Comment:
`final class`?

Reply:
Addressed in https://github.com/aws-amplify/amplify-android/pull/488